profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.features.user_permission.inc

```diff
@@ -0,0 +1,384 @@
+<?php
+/**
+ * @file
+ * nexteuropa_feature_set.features.user_permission.inc
+ */
+
+/**
+ * Implements hook_user_default_permissions().
+ */
+function nexteuropa_feature_set_user_default_permissions() {
```

Comment:
It is in the scope of the ticket, we have to allow cem to access one feature and not any other one. We have to restrict administrator to access that feature, but not any other one